Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -352,6 +352,31 @@ def condition(self, fragment, stepList, key, hasHarvest):
     offset = 0.9,
 )
 
+
+# Track DNN workflows
+class UpgradeWorkflow_trackdnn(UpgradeWorkflow):
+    def setup_(self, step, stepName, stepDict, k, properties):
+        stepDict[stepName][k] = merge([{'--procModifiers': 'trackdnn'}, stepDict[step][k]])
+        custNew = "RecoTracker/FinalTrackSelectors/customiseForDNN.customiseForDNN"
+        if '--customise' in stepDict[stepName][k].keys():
+            stepDict[stepName][k]['--customise'] += ","+custNew
+        else:
+            stepDict[stepName][k]['--customise'] = custNew
```

Comment:
I just realized you are using a customize function inside the modifier. We should not do that - instead, you should remove the customize function completely, and use the modifier to add the needed items in the correct place.